Update to readme.me and Flock docs (#17)
aragon/flock: docs/Proposal guide.md

#### Introductory note
This guide can be used by Flock program applicants in order to draft their ideal proposal. However, the structure and content of the proposal can be adapted as much as necessary by the applicant team. Above all, applications should be **clear and concise**.

Don't forget to share the proposal as early as possible with the [community](https://forum.aragon.org/).

# Aragon Flock proposal: Team name

## 1. Summary

### Strategy

> *This section should detail the high level strategy and rational behind the proposal in simple terms. It can be accompanied by a few short references to past team achievements.*

Reviewer: ```suggestion
*This section should detail the high level strategy and rationale behind the proposal in simple terms. It can be accompanied by a few short references to past team achievements.*
```

*In case the applicant team is an ongoing Flock team it can add the previously achieved roadmap and its most recent community reports.*

### Why us?
*In this part the applicant team should qualitatively explain why it is qualified to deliver the strategy outlined in the previous section.*

### Working in the Aragon Community

*In this section, teams should succintly explain why their time is better spent contributing to Aragon.*

> *Members of the Aragon community are united around the [Aragon Manifesto](https://blog.aragon.org/the-aragon-manifesto-4a21212eac03/) and values of openness, freedom and goodwill.  While there are no requirements to refer to the above explicitly, teams are encouraged to speak from their heart. Technicals are not the only thing!*

Reviewer: ```suggestion
*Members of the Aragon community are united around the [Aragon Manifesto](https://blog.aragon.org/the-aragon-manifesto-4a21212eac03/) and values of openness, freedom, and goodwill.  While there are no requirements to refer to the above explicitly, teams are encouraged to speak from their heart. Technicality is not the only thing that matters!*
```

Note: Flock team ANT package policy is subject to change over time. Please ask the Aragon Association for more information.

## 4. Team

*Below is the basic structure the applicant team should follow when creating a Flock proposal. It includes a brief description the current team members and their relevancy for the role, with links to relevant materials (github, twitter, projects, etc.) and provides info on future openings if the applicant team plans to hire more people during their Flock mandate.*

#### AraEagles Example Team

> - #### John Doe — Project Lead
>
> John has 20 years of experience coding Solidity. He bought Bitcoin in 1995, therefore inventing the blockchain. He admits to be the real Craig Wright.
>
> **Socials**: [GitHub](#) [Twitter](#)
>
> - #### Satoshi Nakamoto — Frontend Dev
>
> Satoshi loves React and React Native. He met John 5 years ago in a hackathon
>
> **Socials**: [GitHub](#) [Twitter](#)
>
> - #### Maria Garcia — Security Researcher
>
> Maria has been working on different projects in the Ethereum both creating and auditing smart contracts. She would like to work full time on pure research.
>
> **Socials**: [GitHub](#) [Twitter](#)

## 5. Requirements

> *List of aragon accounts, channels, tools, assets, domains and infrastructure that you need in order to operate*

Reviewer: ```suggestion
*List of Aragon accounts, channels, tools, assets, domains, and infrastructure that you need in order to operate*
```
